### ISSUE TITLE
embedding size not match

### ISSUE THREAD

**Issue by liu-x-p** [MingjieChen/wavenet_autoencoders#1]
Hi, thank you for sharing your work. I'm learning about unsupervised-speech, and running your code.
I have finished training the WV-VQVAE model, but when I try to run the infer and synthesis code, there would be several error for the size of the loading model is not match with the present model. I changed the model name and some parameters in the hp files, but it still give the error
unexpected key in state_dict: "vq.embedding3.weight" , "vq.embedding4.weight". 
size mismatch for vq.embedding1.weight: copying a param of torch.size([256,64]) from checkpoint, where the shape is torch.size([256,32]) in current model.
size mismatch for vq.embedding2.weight: copying a param of torch.size([256,64]) from checkpoint, where the shape is torch.size([256,32]) in current model.
But I can't find the correspond parameter.
Hope you can help me.
Thank you.

**Comment by MingjieChen** [comment]
I use `hps/infer_hp.json` and `hps/syn_hp.json `as hparams for inference synthesis.

 Please note that you might need to change infer_hp.json or syn_hp.json according to your current model hps.

**Comment by liu-x-p** [comment]
Thanks.
I noticed that you used time-jitter in the code .When I use it, it reports the error: 
_RuntimeError: one of the variables needed for gradient computation has been modified by an inplace operation: [torch.cuda.FloatTensor [64, 64, 1, 16]], which is output 0 of PermuteBackward, is at version 4; expected version 0 instead. Hint: the backtrace further above shows the operation that failed to compute its gradient. The variable in question was changed in there or anywhere later. Good luck!_

I checked the code, and it happens when I set the parameter use-time-jitter as true. I have changed the inplace operation `x[:,:,1] = x[:,:,0]` to be `x1[:,:,1] = x[:,:,0]`, but it doesn't work. How can I solve it?

**Comment by MingjieChen** [comment]
Hi, I haven't test the time jitter part of code thoroughly, 
I just implemented as my own understanding.

I am working on other projects currently, and not have enough time to test the time jitter for now. 

Pull request is welcome
Thanks

**Comment by liu-x-p** [comment]
Thank you for your reply!
